style.css

```diff
@@ -0,0 +1,136 @@
+*{
+    margin:0;
+    padding:0;
+}
+header img{
+    height: 70px;
+    margin-left: 10%;
+}
+section header{
+    height: 30vh;
+    background-color: #171A29;
+}
+section header img{
+    float: left;
+    margin: 3vh 0 3vh 12%;
```

Comment:
As mentioned above, the gutter space should come from the parent. Fix this here as well and at all other places wherever applicable.